tools/covert_csv2jsonl.py

```diff
@@ -58,9 +59,12 @@ def get_paths(self) -> List[str]:
   def parse_csv(self, file_paths: List[str]) -> None:
     """Parse all csv file to get company related field
     """
-    print("=== Read CSV files... ==="")
+    print("=== Read CSV files... ===")
     for path in tqdm(file_paths):
-      df = pd.read_csv(path, names=self.column_names)
+      
+      with codecs.open(path, "r", "Shift-JIS", "ignore") as file:
+        df = pd.read_table(file, names=self.column_names,delimiter=",")
```

Comment:
I assume that users should download the [Unicode format](https://www.houjin-bangou.nta.go.jp/download/zenken/#csv-unicode) data instead of Shift-JIS format. If so, the error should disappear. 

If using the Unicde CSV files and hard encoding the ''Shift-JIS" format, the text will become illegal characters.
```
>>> df['cityName'][:5]
>>> 蜊莉｣逕ｰ蛹ｺ\n1    蜊莉｣逕ｰ蛹ｺ\n2    蜊莉｣逕ｰ蛹ｺ\n3    蜊莉｣逕ｰ蛹ｺ\n4    蜊莉｣逕ｰ蛹ｺ\n
```

So I recommend you download the Unicode format data as the instruction in README. 
For this commit, you could delete these two lines.